analysis_options.yaml

```diff
@@ -20,6 +20,9 @@ analyzer:
     non_constant_identifier_names: ignore
     # Consider removing from dart_flutter_team_lints.
     unreachable_from_main: ignore
+    # MDN comments include HTML docs that trigger this lint. Reenable once
+    # that's resolved.
```

Comment:
nit: should have a bug to reference here?

Reply:
https://github.com/dart-lang/sdk/issues/56450 Done!